packages/ord-connect/src/lib/signMessage.ts

```diff
@@ -0,0 +1,40 @@
+import { ordit } from "@sadoprotocol/ordit-sdk";
+import { Network, Wallet } from "../providers/OrdContext.tsx";
+
+interface SignMessageParams {
+  message: string;
+  wallet: Wallet;
+  address: string | Record<string, string>;
+  network: Network;
+}
+
+export default async function getSignedMessage({
+  message,
+  wallet,
+  address,
+  network,
+}: SignMessageParams) {
+  if (wallet === Wallet.UNISAT) {
+    const signedMessage = await ordit.unisat.signMessage(message);
```

Comment:
Is there any flow with errors? Like if user rejects? Should this whole function be wrapped in try/catch?

Reply:
Ok I saw that this function is used in hook with try/catch, so this comment can be ignored.